.github/dependabot.yml

```diff
@@ -18,3 +18,45 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types: ["version-update:semver-minor"]
+    groups:
+      patches:
+        applies-to: version-updates
+        patterns:
+          - "*"
```

Comment:
I don't think this is required, based on the examples in the docs — omitting `patterns` should be the same as having `*` as the only pattern.

Reply:
O gotcha. Was being a bit too cautious.